explainer.md

```diff
@@ -687,13 +710,18 @@ enum VRFrameOfReferenceType {
 
 interface VRFrameOfReference : VRCoordinateSystem {
   readonly attribute VRStageBounds? bounds;
+  attribute EventHandler onboundschange;
 };
 
 interface VRStageBounds {
-  readonly attribute double minX;
-  readonly attribute double maxX;
-  readonly attribute double minZ;
-  readonly attribute double maxZ;
+  readonly attribute FrozenArray<VRStageBoundsPoint> geometry;
+  readonly attribute VRStageBoundsPoint playAreaMin;
+  readonly attribute VRStageBoundsPoint playAreaMax;
+};
+
+interface VRStageBoundsPoint {
+  readonly attribute double x;
+  readonly attribute double z;
```

Comment:
In general, I think describing a 2D point as x,y is safe, even though we know it represents location on the xz plane. No matter what we choose we will confuse some users, but I think x,z will confuse more users than x,y in this case.